Resources/public/js/views/services/ez-sectionserversideviewservice.js

```diff
@@ -39,6 +39,14 @@ YUI.add('ez-sectionserversideviewservice', function (Y) {
                 tasks = new Y.Parallel(),
                 contentService = this.get('capi').getContentService();
 
+            this.fire('notify', {
+                notification: {
+                    identifier: 'assign-section',
```

Comment:
you should add a method to generate the identifier so that you don't repeat it (I would even create a method to fire the event as well) and in addition, the identifier should be more *unique* so that if the user assign a section to others contents, he gets another notification bar. So you should include the section id and the content ids in it.